### DISCUSSION TITLE
Sources: Programs: Type explanation

### DISCUSSION THREAD

**Discussion by Iliaroz** [goldendict/goldendict#1641] [Q&A] [answered]
Can somebody explain me what is the "type" in Dictionaries -> Sources -> Programs ???

I understand "html" and "plain text", but what  are "audio" and "prefix match" what expected by GoldenDict?

**Comment by Abs62** [comment, marked as answer]
>> but what are "audio"

GD insert to article link to program which should work as link to audio file.

>> and "prefix match" what expected by GoldenDict?

GD expect list of words matched with word passed to program.

**Reply by Iliaroz** [reply]
Thank you for your quick answer.

Can you give a bit more explanation of "prefix match" option?
in which form the list should be formed? coma-separated, space-separated or something else?

**Comment by Abs62** [comment]
[CR]LF-separated. One word per row.
